src/screens/TeamMatchup.tsx

```diff
@@ -0,0 +1,19 @@
+import { Flex } from "@chakra-ui/react";
+import { useLocation } from "react-router-dom";
+
+const TeamMatchup: React.FC = () => {
+  const location = useLocation();
+  const searchParams = new URLSearchParams(location.search);
+  const team1 = searchParams.get("team1");
+  const team2 = searchParams.get("team2");
```

Comment:
After you make the change to the router, this can be
```
const { matchupTeams } = useParams();

 const [team1, team2] = matchupTeams.split('-');
```